src/server/pfs/s3/object.go

```diff
@@ -124,6 +164,26 @@ func (c *controller) CopyObject(r *http.Request, srcBucketName, srcFile string,
 	return version, nil
 }
 
+func (lfs *localFS) copyObject(pc *client.APIClient, srcBucket, destBucket *Bucket, srcFile, destFile string) (string, error) {
+	src, err := os.Open(filepath.Join(srcBucket.Path, srcFile))
+	if err != nil {
```

Comment:
Do we need to rewire the errors here so that the client function can make them PFS like?